src/style.css

```diff
@@ -0,0 +1,79 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: Avenir;
```

Comment:
add fallback font here

Reply:
check this [guide](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) (7 point)